### ISSUE TITLE
Hi, I see that your wip logo is gone. Can you train the soundstream (loss drops normally) and achieve good results?

### ISSUE THREAD

**Issue by zjj1333** [lucidrains/audiolm-pytorch#26]
Hi, I see that your wip logo is gone. Can you train the soundstream (loss drops normally) and achieve good results?

**Comment by eonglints** [comment]
Hey, I'll be having a look at this next week. I'll let you know how I get on! At this stage I believe @lucidrains is predominantly focused on setting up the architectures.

**Comment by erogol** [comment]
FYI I've tried training it with a single batch to overfit but it didn't. I am also casually checking the code and I'll share if I find something.

**Comment by lucidrains** [comment]
yea people are successfully training it now
